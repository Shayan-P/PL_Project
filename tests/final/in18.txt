def f(n = 2):
    global a;
    a = 10;
    print(8);
    return 17;
;

a = 5;
b = f(18);
print(b);
print(a);