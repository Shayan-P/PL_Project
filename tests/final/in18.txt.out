8
17
5
'finished
